app/quadraticlands/management/commands/pull_steward_info.py

```diff
@@ -0,0 +1,55 @@
+    GTC_CONTRCT_ADDRESS = Web3.toChecksumAddress('0xde30da39c46104798bb5aa3fe8b9e0e1f348163f')
+    return web3.eth.contract(address=GTC_CONTRCT_ADDRESS, abi=ABI)
+
+    return _poh_contract
+
+
+class Command(BaseCommand):
+
+    help = 'pulls all gitcoin steward metadata info'
+
+    def handle(self, *args, **options):
+        web3 = get_web3('mainnet')
+        gtc = get_gtc_contract(web3)
+        print("votes_delegated, handle")
+        for gtcs in GTCSteward.objects.all():
+            try:
+                votes = gtc.functions.getCurrentVotes(Web3.toChecksumAddress(gtcs.gtc_address)).call()
+                delegates = gtc.functions.delegates(Web3.toChecksumAddress(gtcs.gtc_address)).call()
+                votes = round(votes / 10 **18)
+                print(f"{votes},{gtcs.profile.handle}")
+            except Exception as e:
+                pass;
+
```

Comment:
^ on a sidenote: Hmm a nicer to do here would be.

-  add this info stewards model 
- create new steward auto-matically
- fetch from gitcoin.gov their steward bio

This would roll up into scott's request on making the stewards report card.